.github/workflows/ci.yml

```diff
@@ -0,0 +1,97 @@
+# Builds Betaflight Configurator on Windows, Android, Linux and macOS platforms.
+#
+# After building, artifacts are released to a seperate repository.
+
+name: CI
+
+on:
+  push:
+    branches:
+      - master
+      - '*-maintenance'
+  pull_request:
+    branches:
+      - master
+      - '*-maintenance'
+
+jobs:
+  test:
+    name: Test
+    runs-on: ubuntu-20.04
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Install Node.js 14.18
```

Comment:
14.18.2 is LTS

Reply:
I suppose 14.18 is the LTS. If some bug is found in the LTS version a 14.18.3 will be released. Another thing is that if we can use the same version that we publish in:

https://github.com/betaflight/betaflight-configurator/blob/e43025cffe50eb1453f2da7977506f9a6064c8ff/.nvmrc#L1

but this implies to remember to modify both files at the same time.

Reply:
Yes 14.18 is the same as 14.18.x, so it will get the latest patch. The LTS is simply Node 14 (same as 14.x.x) with minor version updates as well. I didn't know you used nvm, i will look into using that version instead. However, unless we have a good reason for locking even the patch version, I would consider using at least 14.18.x since patches are non-breaking and could be important bug/security fixes.

Reply:
Thanks for the heads up

Reply:
Right, it can make sense to update the version manually for reproducible builds, but this node.js version is not a dependency of the executable but a tool used to "compile" it and patches are non-breaking and should not change the output of the build (but may change how it's build). Anyway I don't really mind fixing the version. I have updated PR to use .nvmrc version.